props/saber_fett263_buttons.h

```diff
@@ -687,103 +689,20 @@ class SoundLibrary {
   void SaySwingColor() { Play("mswing.wav"); }
   void SaySwingIgnition() { Play("mswingon.wav"); }
   void SaySwingOnSpeed() { Play("mswingsp.wav"); }
-  void SayThrustIgnition() { Play("mthrston..wav"); }
+  void SayThrustIgnition() { Play("mthrston.wav"); }
   void SayTwistIgnition() { Play("mtwiston.wav"); }
   void SayTwistRetraction() { Play("mtwstoff.wav"); }
-  void SayUp() { Play("mup.wav"); }
+  void SayUp() { Play("mup.wav"); } // Sound for increase
   void SayVolumeMenuEnd() { Play("vmend.wav"); }
-  void SayVolumeUp() { Play("volup.wav"); }
-  void SayZoom() { Play("mzoom.wav"); } // Zoom in for color menu "zooming in"
-  void SayVolumeDown() { Play("voldown.wav"); }
+  void SayVolumeUp() { Play("volup.wav"); } // Sound for increasing volume
+  void SayZoom() { Play("mzoom.wav"); } // Sound for color menu "zooming in"
```

Comment:
If that's what it says, we should probably rename this function to SayZoomingIn()
The intention is that the function name should be as close to the actual sound as possible.

Reply:
updated